src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/RazorHtmlCSharpLanguageServer.cs

```diff
@@ -251,6 +252,42 @@ public Task<DiagnosticReport[]> DocumentPullDiagnosticsAsync(DocumentDiagnostics
             return handler.HandleRequestAsync(request, clientCapabilities, cancellationToken);
         }
 
+        private static JsonRpc CreateJsonRpc(Stream inputStream, Stream outputStream, object target)
```

Comment:
This is the smallest of nitpicks, but why invert the order of input and output considering they're the other way around when you use them?

Reply:
Not smallest of nitpicks at all. Totally valid. I'll change